### ISSUE TITLE
Problem in testing my own video

### ISSUE THREAD

**Issue by lan786** [akanazawa/human_dynamics#46]
Hi @akanazawa  First of all thanks for your this kind work. It's really helpful. I'm facing following problem... 
1. This work is working fine with your provided videos using pre-trained weights. But when I run it on random video (single person), all the process runs successfully till PoseFlow successfully ran! but it stuck on Processing frames. 
2. I checked the demo_output folder --> video_folder and I found that there are only two folders inside it, one is AlphaPose_output and other one is video_frames. so no other folders like hmmr_output and hmmr_output_crop. 
3. In AlphaPose_output folder there are images frames of video along with json files. 
4. When I run it with your videos images frames are not stored within AlphaPose_output folder and all files are in separate folders as specified in project code. I'm unable to understand that what is problem with any wild video.  Code it ok this problem is just with new video.
5: Moreover I checked json files generated by your videos and my video... Json files seem to be ok.

**Comment by jasonyzhang** [comment]
Hi,

I suspect the issue might result from a lack of person detections in the first frame. The easiest way to debug this though is by uploading the video you are trying to test on.

Best,
Jason

**Comment by lan786** [comment]
@jasonyzhang  Hi thanks for your reply. Person detection in video is fine. Alphapose is working fine. I checked my video with AlphasPose and it seems to be ok. Did you check any wild video with this project? If yes then results are ok?  If possible then can you provide me any video I just want to compare frames.

**Comment by jasonyzhang** [comment]
Yes, we've tested on in-the-wild videos. Most of the demo videos are scraped from Instagram. Could you upload the video that you are having trouble with?

**Comment by lisa676** [comment]
@jasonyzhang Hi Jason, I tested with TikTok and YouTube videos. I've sent you an email with tested videos and related problem. Kindly check your email and suggest some solutions. It will be more good if you will test my videos by yourself in your project.
